lib/validators/dependencies.js

```diff
@@ -12,10 +12,10 @@ module.exports = function dependencies(schema, tpl) {
     const value = schema.dependencies[dependency];
     const error = tpl.error('dependencies');
 
-    tpl(`if (${tpl.data}.hasOwnProperty(unescape("${escape(dependency)}"))) {`);
+    tpl(`if (${tpl.data}.hasOwnProperty(decodeURIComponent("${escape(dependency)}"))) {`);
     if (Array.isArray(value) || typeof value === 'string') {
       [...value]
-        .map(property => `if (!${tpl.data}.hasOwnProperty(unescape("${escape(property)}"))) ${error}`)
+        .map(property => `if (!${tpl.data}.hasOwnProperty(decodeURIComponent("${escape(property)}"))) ${error}`)
```

Comment:
escape -> `escapeURIComponent`?

Reply:
Replaced "escape" at all and it still works )). Please pull once more.